components/rpill/browser/rpill.cc

```diff
@@ -0,0 +1,32 @@
+/* Copyright 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+#include <locale>
+#include <string>
+
+#include "brave/components/rpill/browser/rpill.h"
+
+#include "base/strings/string_util.h"
```

Comment:
this doesn't look like it is used?

Reply:
It is required for `base::ToLowerASCII()` https://chromium.googlesource.com/chromium/src/+/edce9a33027cc5f73c4866d70e34f690f6720a56/base/strings/string_util.h#96